_data/versions.yml

```diff
@@ -1394,6 +1394,42 @@ v3.0:
 
   components:
     felix:
+      version: 3.0.0-alpha1
+      url: https://github.com/projectcalico/felix/releases/tag/3.0.0-alpha1
+    typha:
+      version: v0.6.0-alpha1
+      url: https://github.com/projectcalico/typha/releases/tag/v0.6.0-alpha1
+    calicoctl:
+      version: v2.0.0-alpha1-rc2
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v2.0.0-alpha1-rc2
```

Comment:
this should be `https://github.com/projectcalico/calicoctl/releases/tag/v2.0.0-alpha1`